autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -26,12 +27,20 @@ const NavBar: React.SFC = () => {
         color="primary"
       >
         <Toolbar>
-          <Typography
-            variant="h6"
-            className={classes.title}
-          >
-            Auto Scheduler
-          </Typography>
+          <div className={classes.title}>
+            <Button
```

Comment:
```suggestion
            <Button
              classes={{ label: classes.buttonText }}
```

I'm not a fan of the all-uppercase text. If you add a field called `buttonText` to the `makeStyles()` above, you can tell the button to apply that class to the label of the button. In that field, set `textTransform` to 'capitalize', since the default is 'uppercase'

Reply:
Oh I didn't even notice it was all caps lmao, I like this much better